generator/scripts/generateInputRange.py

```diff
@@ -0,0 +1,97 @@
+import openpyxl  # Read data from Excel
```

Comment:
Add some doc on top to explain how this is meant to be used. Is it a library to include in the generator script?

Reply:
@tahini I'm not sure I understand what you have in mind.